src/main/java/at/hannibal2/skyhanni/utils/SkyBlockItemModifierUtils.kt

```diff
@@ -47,7 +47,7 @@ object SkyBlockItemModifierUtils {
 
     fun ItemStack.getPetItem(): String? {
         val data = cachedData
-        if (data.heldItem == "") {
+        if (data.heldItem == "" || data.heldItem == null) {
```

Comment:
null assumes no pet item loaded. we would check each time this way, making the cache unnecessary